server/services/UsersService.js

```diff
@@ -213,7 +213,7 @@ async function getAllTesters() {
 }
```

Comment:
There was a bug in signup. It was not propagating the user id to the `req.session.user`

Reply:
ahh thanks for catching